Jonathanlau92/prayers-letterbox
# Allow editing of prayers and restrict access

# Pull Request Description 

Task(s) you did:
- Allow editing of prayers
- Allow deleting of prayers (Deleting prayers will delete categories)
- Restrict access to prayers. Only current_user will have access to their own edit and delete button. If manually typing in the URL, pundit gem will return an error message as shown below.
<img width="1437" alt="Screenshot 2020-07-18 at 11 38 16 PM" src="https://user-images.githubusercontent.com/40416736/87856195-0b28a080-c950-11ea-9b06-59577291fda1.png">



How did you test it? : 
- Tested by trying out the edit and delete of prayers
- Ensure that you cant see edit and delete button of other users
- Ensure that pundit restricts manual url typing to edit other user's prayers


Remarks:
